edge-patron/src/main/resources/domain/edge-patron/features/util/initData.feature

```diff
@@ -0,0 +1,262 @@
+Feature: init data for edge-patron
+
+  Background:
+    * url baseUrl
+    * callonce login { tenant: 'diku', name: 'diku_admin', password: 'admin' }
+    * def headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+
+  @postMaterialType
+  Scenario: create materialtype
```

Comment:
Please format these scenario names consistently (with words separated by spaces and either capitalised or not)

```suggestion
  Scenario: create material type
```